tests/test_unix.py

```diff
@@ -7,15 +7,46 @@
 from _pytest.monkeypatch import MonkeyPatch
 from pytest_mock import MockerFixture
 
+import platformdirs.unix
 from platformdirs.unix import Unix
 
 
+def test_user_documents_dir(monkeypatch: MonkeyPatch) -> None:
+    example_path = "/home/example/ExampleDocumentsFolder"
+    monkeypatch.setattr(platformdirs.unix, "get_user_dirs_folder", lambda key: example_path)
+    assert Unix().user_documents_dir == example_path
+
+
+def test_user_documents_dir_env_var(monkeypatch: MonkeyPatch) -> None:
+    # Mock documents dir not being in user-dirs.dirs file
+    monkeypatch.setattr(platformdirs.unix, "get_user_dirs_folder", lambda key: None)
+
+    example_path = "/home/example/ExampleDocumentsFolder"
+    monkeypatch.setenv("XDG_DOCUMENTS_DIR", example_path)
+
+    assert Unix().user_documents_dir == example_path
+
+
+def test_user_documents_dir_default(monkeypatch: MonkeyPatch) -> None:
+    # Mock documents dir not being in user-dirs.dirs file
+    monkeypatch.setattr(platformdirs.unix, "get_user_dirs_folder", lambda key: None)
```

Comment:
can you use here mocker instead? and replace monkeypatch.setenv with mocker.patch.dict on os.environ